### ISSUE TITLE
Can't run project NextJS example

### ISSUE THREAD

**Issue by thinhnguyen1105** [celo-org/use-contractkit#84]
I found the project example written by NextJS inside the package of the project. I got that example and try to run it but was not successful.
The error here: Can't resolve '@celo-tools/use-contractkit' with "@celo-tools/use-contractkit": "2.0.0-dev", inside package.json
I need use-contractkit work with nextjs. Please help

**Comment by nicolasbrugneaux** [comment]
Hi @thinhnguyen1105 :) 

If you want to use `use-contractkit` I'd recommend bootstrapping your own nextjs app and importing the [production version](https://www.npmjs.com/package/@celo-tools/use-contractkit/v/2.1.0) of the library. 
You can do this by following [this tutorial](https://nextjs.org/docs/getting-started) and then importing use-contractkit like this: 
```sh
yarn add @celo-tools/use-contractkit
``` 
and then importing it in your nextjs app like this 
```ts
import { useContractKit } from '@celo-tools/use-contractkit';
```


---

However, if you _really_ want to try playing around with the example app in the monorepo, you'll need to run these steps:

```sh
# from the root of the monorepo run these 2 commands:
yarn # to install dependencies
make dev # to compile/watch the library and start the development server of the example app
```

If you run into that error message: `example: Module not found: Can't resolve '@celo-tools/use-contractkit/lib/styles.css'`, it is a known issue of the first `make dev` run, so just run CTRL+C the current `make dev` process, and run it again, it should run fine the second time around.

Please let us know if you managed to do either solutions

**Comment by thinhnguyen1105** [comment]
Still error when run script `make dev`. And when i use @celo-tools/use-contractkit 2.1.0 for my dapp. Get another error "Uncaught TypeError: Class extends value undefined is not a constructor or null" in walletconnect dependency

**Comment by nicolasbrugneaux** [comment]
Sorry this issue went under my radar, apologies! Do you still encounter this issue? This should have been fixed by the 2.1.12 release on the 30th of December.

**Comment by nicolasbrugneaux** [comment]
Closing this as we have no news, feel free to re-open if the issue still exists.
